src/lib/profiles/data-management/Current/SubscriptionClient.cpp

```diff
@@ -2114,6 +2120,12 @@ void SubscriptionClient::OnUpdateTimerCallback(System::Layer * aSystemLayer, voi
     pClient->UpdateTimerEventHandler();
 }
 
+void SubscriptionClient::OnSubscribeScheduleWorkCallback(System::Layer * aSystemLayer, void * aAppState, System::Error)
```

Comment:
This needs to be pulled outside the `#if WEAVE_CONFIG_ENABLE_WDM_UPDATE` in line 2064 above

Reply:
Yep, good catch, just fixed it, thanks

Reply:
thanks!